sandbox/src/components/renderingZone.tsx

```diff
@@ -22,8 +24,18 @@ interface IRenderingZoneProps {
     expanded: boolean;
 }
 
+function isTextureAsset(name: string): boolean {

```

Comment:
This will fail if the texture is loaded with query param (i.e. `https://url/image.jpg?atag=123456`). Are those being filtered? might be safer to use contains rather than ends with

Reply:
(note - if it is just for drag-and-drop it should be perfectly fine the way it is)

Reply:
I suppose it could have query param. It's an unlikely scenario though. I'll check for the `?`. Using contains isn't perfect either since the query param could have `.<ext>` in them.